pkg/antctl/raw/check/constants.go

```diff
@@ -15,5 +15,5 @@
 package check
 
 const (
-	DefaultTestImage = "antrea/toolbox:1.4-0"
+	DefaultTestImage = "antrea/toolbox:1.5-1"
```

Comment:
Is the toolbox version upgrade necessary for this PR?

Reply:
No, I will revert this change, thanks.